KillTeam/ViewModels/ModelsViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.ObjectModel;
+
+namespace KillTeam.ViewModels
+{
+    public class ModelsViewModel : ObservableCollection<ModelsProfileViewModel>
```

Comment:
Same with this, should it be `MemberViewModel` instead of `ModelsViewModel`?

Reply:
This is used to display actual models (before they because members)

Reply:
Yeah I get that, but to avoid confusion (oh look we've already caused some) IMO we should change the name to Member, as they are a member of the Kill Team and avoids confusion with the term model in our architecture.

Reply:
You’re right, let’s change it ;)